src/main/java/ca/corefacility/bioinformatics/irida/ria/config/thymeleaf/webpacker/util/WebpackEntryDeserializer.java

```diff
@@ -0,0 +1,65 @@
+package ca.corefacility.bioinformatics.irida.ria.config.thymeleaf.webpacker.util;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import ca.corefacility.bioinformatics.irida.exceptions.WebpackParserException;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+
+/**
+ * Deserializer for the webpack manifest file entry.
+ */
+public class WebpackEntryDeserializer extends StdDeserializer<WebpackEntry> {
+
+	public WebpackEntryDeserializer() {
+		this(WebpackEntry.class);
+	}
+
+	public WebpackEntryDeserializer(Class<?> vc) {
+		super(vc);
+	}
+
+	@Override
+	public WebpackEntry deserialize(JsonParser jsonParser, DeserializationContext deserializationContext)
+			throws IOException, JsonProcessingException {
```

Comment:
I have a warning here that `JsonProcessingException` is never thrown.  It's a subclass of `IOException` so the `JsonProcessingException` can be removed here.

Reply:
Updated in 324a942